Iniciante/2126/2126.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+    int i, j, casos, contador, k, l;
+    char subsq[100], seq[100];
+
+    casos = 0;
+
+    while(scanf("%s%s", subsq, seq)!=EOF)
+    {
+        int total, posicao, tmp, tam_subsq, tam_seq;
+
+        tam_subsq = strlen(subsq); 
+        tam_seq = strlen(str);
```

Comment:
```suggestion
        tam_seq = strlen(seq);
```

tava causando erro de compilação